MockKitUI/Assets.xcassets/IAGC.imageset/Contents.json

```diff
@@ -1,8 +1,8 @@
 {
   "images" : [
     {
+      "filename" : "IAGC.pdf",
```

Comment:
[nit] Perhaps `mockPump`?

Reply:
these were the names of the files as given to me by Paul.

Reply:
Same, this mock pump is not an implementation of IAGC or representative of IAGC in any way. It also doesn't look like a pump? Could we get design to take another crack at this? It should have a port, or something that at least hints at being pump-like.